components/dashboard/components/NetworkGraph/index.jsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2018-present Riff Learning, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable header/header */
+
+import lifecycle from 'react-pure-lifecycle';
+import {connect} from 'react-redux';
+import {Datasets as MMUserAnalyticsDatasets} from 'mattermost-redux/constants/user_analytics';
+
+import * as UserActions from 'actions/user_actions.jsx';
+
+import UserStore from 'stores/user_store.jsx';
+import TeamStore from 'stores/team_store.jsx';
+
+import {NetworkGraphView} from './NetworkGraphView';
+
+const mapStateToProps = () => ({
+    userInteractions: UserStore.getUserAnalytics()[MMUserAnalyticsDatasets.USER_INTERACTIONS],
+    userLearningGroups: UserStore.getUserAnalytics()[MMUserAnalyticsDatasets.USER_LEARNING_GROUPS],
+    learningGroupTypes: UserStore.getUserAnalytics()[MMUserAnalyticsDatasets.LEARNING_GROUP_TYPES],
```

Comment:
Probably a silly question, but didn't you write an action in `user_actions` to wrap the calls to `UserStore.getUserAnalytics()`? It's not obvious to me why we don't use that action here instead of making these calls directly.